src/const_eval.md

```diff
@@ -23,7 +23,7 @@ to be run.
 * [Const parameters].
 * [Paths] to [functions] and [constants].
   Recursively defining constants is not allowed.
-* Paths to [statics]. These are only allowed within the initializer of a static.
+* Paths to immutable [statics].
```

Comment:
You can use the path but evaluation will fail. Any attempt to read from a mutable or interior mutable static is an error.

However, if this is dead code, it will get accepted. So this is different from the prior check that just rejected even syntactically mentioning statics.

Reply:
Oh, sorry, my testing is wrong because these errors don't happen with `cargo check` 😦 .

Reply:
`const { ... }` blocks don't get evaluated in `cargo check`, yeah. `const` *items* do.